### PR TITLE
Update bitbucket-server.mdx

### PR DESCRIPTION
**Pull request by kristinahunter**
BitBucket updated their software to add OAuth 2. The supported method for integrating BitBucket Server with Terraform Cloud today is using OAUTH 1, so different steps have to be taken to set this up

### Screenshots

----------

### Merge Checklist
_If items do not apply to your changes, add (N/A) and mark them as complete._

#### Pull Request
- [ ] One or more labels describe the type of change (e.g. clarification) and associated product (e.g. tfc).
- [ ] Description links to related pull requests or issues, if any.

#### Content
- [ ] Redirects have been added for moved, renamed, or deleted pages. This requires a separate PR in the [`terraform-website` repository](https://github.com/hashicorp/terraform-website) `redirects.next.js` file.
- [ ] API documentation and the API Changelog have been updated. 
- [ ] Links to related content where appropriate (e.g., API endpoints, permissions, etc.).
- [ ] Pages with related content are updated and link to this content when appropriate.
- [ ] Sidebar navigation files have been updated for added, deleted, reordered, or renamed pages.
- [ ] New pages have metadata (page name and description) at the top.
- [ ] New images are 2048 px wide. They have HashiCorp standard annotation color (#F92672) and format (rectangle with rounded corners), blurred sensitive details (e.g. credentials, usernames, user icons), and descriptive alt text in the markdown for accessibility.
- [ ] New code blocks have the correct syntax and line breaks to eliminate horizontal scroll bars.
- [ ] UI elements (button names, page names, etc.) are bolded.
- [ ] The Vercel website preview successfully deployed.

#### Reviews
- [ ] I or someone else reviewed the content for technical accuracy.
- [ ] I or someone else reviewed the content for typos, punctuation, spelling, and grammar.

### CODE REVIEW

File: website/docs/cloud-docs/vcs/bitbucket-server.mdx [hunk 1]
```diff
@@ -91,7 +91,9 @@ Leave the page open in a browser tab, and remain logged in as an admin user.
 
    ![Bitbucket Server screenshot: The application links page](/img/docs/bitbucket-server-application-links.png)
 
-1. Enter Terraform Cloud's URL in the text field (`https://app.terraform.io`, or the hostname of your Terraform Enterprise instance) and click the "Create new link" button.
+1. In application links, select Create link.
+2. Select Atlassian product as the link type. This option also works for external applications. This is the original application links mechanism – Atlassian has left it under this option so users can still use their OAuth 1.0 integrations.
```
Comment (mgarrell777) [inline website/docs/cloud-docs/vcs/bitbucket-server.mdx:95]: I suggest rewording to "This method is the original application links mechanism. Atlassian has left it under this option so users can still use their OAuth 1.0 integrations.

**Review by mgarrell777 — APPROVED** [review]
Hi @kristinahunter I'm Matthew, new technical writer with the Terraform team. Very good stuff here. I approve with the 1 very tiny wording change suggestion.

Thanks very much for your time and effort.

File: website/docs/cloud-docs/vcs/bitbucket-server.mdx (hunk 1 above)
Comment (laurapacilio) [inline website/docs/cloud-docs/vcs/bitbucket-server.mdx:95]: I'm wondering if we can shorten that even more. I want to phrase this in terms of what this lets users do and address the user as "you" rather than talking about "users" more broadly. Thoughts? 

@mgarrell777 - Feel free to weigh in here too! I took you original edit above and put it here. :) 
```suggestion
1. Select **Create link** in application links.
2. Select **Atlassian product** as the link type. This option also works for external applications lets you continue to use OAuth 1.0 integrations.
```

**Review by laurapacilio — APPROVED** [review]
I don't want to block you on this, but I left a suggestion (that expands upon Matthew's original suggestion!) for your review. Let me know what you think. Thank you for adding this!

**Comment by ggambetti** [comment]
👋 Taking over from @kristinahunter.

**Comment by ggambetti** [comment]
@mgarrell777 👋 Let me know if you have any comments, otherwise I'm planning on merging this week.

File: website/docs/cloud-docs/vcs/bitbucket-server.mdx [hunk 2]
```diff
@@ -91,7 +91,9 @@ Leave the page open in a browser tab, and remain logged in as an admin user.
 
    ![Bitbucket Server screenshot: The application links page](/img/docs/bitbucket-server-application-links.png)
 
-1. Enter Terraform Cloud's URL in the text field (`https://app.terraform.io`, or the hostname of your Terraform Enterprise instance) and click the "Create new link" button.
+1. Select **Create link** in application links.
+2. Select **Atlassian product** as the link type. This option also works for external applications and lets you continue to use OAuth 1.0 integrations.
+3. Enter Terraform Cloud's URL in the text field (https://app.terraform.io, or the hostname of your Terraform Enterprise instance).
```
Comment (mgarrell777) [inline website/docs/cloud-docs/vcs/bitbucket-server.mdx:96]:
```suggestion
3. Enter Terraform Cloud's URL in the text field (https://app.terraform.io or the hostname of your Terraform Enterprise instance).
```

Removal of an unneeded comma

File: website/docs/cloud-docs/vcs/bitbucket-server.mdx [hunk 3]
```diff
@@ -91,7 +91,9 @@ Leave the page open in a browser tab, and remain logged in as an admin user.
...
 
    ~> **Note:** If you're connecting multiple Terraform Cloud organizations to the same Bitbucket Server instance, you can only use Terraform Cloud's main URL once. For subsequent organizations, you can enter the organization URL instead. Organization URLs look like `https://app.terraform.io/app/<ORG NAME>` or `https://<TFE HOSTNAME>/app/<ORG NAME>` — it's the page Terraform Cloud's "Workspaces" button takes you to.
```
Comment (mgarrell777) [inline website/docs/cloud-docs/vcs/bitbucket-server.mdx:98]:
```suggestion
   ~> **Note:** If you connect multiple Terraform Cloud organizations to the same Bitbucket Server instance, you can only use Terraform Cloud's main URL once. For subsequent organizations, you can enter the organization URL instead. Organization URLs are the pages you go to using Terraform Cloud's **Workspaces** button and look like `https://app.terraform.io/app/<ORG NAME>` or `https://<TFE HOSTNAME>/app/<ORG NAME>`.
```

Suggest rewording these sentences for clarity and to avoid contractions.
